fuel-core/src/service/modules.rs

```diff
@@ -61,25 +65,24 @@ pub async fn start_modules(config: &Config, database: &Database) -> Result<Modul
         .db(Box::new(database.clone()) as Box<dyn TxPoolDb>)
         .import_block_event(block_importer.subscribe());
 
-    let p2p_mpsc = ();
-    let p2p_broadcast_consensus = ();
-    let p2p_broadcast_block = ();
+    let (tx_request_event, rx_request_event) = mpsc::channel(100);
```

Comment:
Is there a reason these channels are set at 100? I think this may have similar concerns as @Voxelot 's comment here https://github.com/FuelLabs/fuel-core/pull/491#discussion_r931550844

Reply:
his concern was mainly because no one would be consuming those messages on the other end, in this case there is the consumer `rx_request_event` so it's fine.